src/containers/AccessPointDetails/components/Status/index.js

```diff
@@ -62,27 +119,33 @@ const Status = ({ data }) => {
 
           {renderSpanItem(
             'Noise Floor',
-            data.status &&
-              data.status.radioUtilization &&
-              data.status.radioUtilization.detailsJSON &&
-              data.status.radioUtilization.detailsJSON.avgNoiseFloor
+            checkNoise(
+              data.status &&
+                data.status.radioUtilization &&
+                data.status.radioUtilization.detailsJSON &&
+                data.status.radioUtilization.detailsJSON.avgNoiseFloor
```

Comment:
change to use optional chaining ` data?.status?.radioUtilization?.detailsJSON?.avgNoiseFloor`